src/main/java/prestudy/team4/board/domain/Comment.java

```diff
@@ -3,23 +3,33 @@
 
 import jakarta.persistence.*;
 import lombok.*;
-
 import java.time.LocalDateTime;
 
 @Getter
 @NoArgsConstructor @AllArgsConstructor @Builder
 @Entity
+@Table(name = "comments")
 public class Comment {
 
     @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+    private Long commentId;
 
-    // 임시: 나중에 Post, User 엔티티 완성되면 @ManyToOne으로 교체
+    @Column(nullable = false)
     private Long postId;
```

Comment:
Comment 테이블이 Post와 User 테이블에 연결되어 있지 않아요.
현재는 `postId`와 `userId`가 그저 숫자로 저장됩니다.
이러면 게시글이 삭제되었을 때 댓글은 DB에 그대로 남게 되어서 문제가 돼요.
`fetch = FetchType.LAZY`로 성능 최적화가 가능해요.
```suggestion
    @ManyToOne(fetch = FetchType.LAZY) // 댓글(N) : 게시글(1) 관계
    @JoinColumn(name = "post_id", nullable = false) // Post 테이블의 post_id 컬럼과 매핑
    private Post post;
    
    @ManyToOne(fetch = FetchType.LAZY) // 댓글(N) : 사용자(1) 관계
    @JoinColumn(name = "user_id", nullable = false) // User 테이블의 user_id 컬럼과 매핑
    private User user;
```